notebooks/community/ml_ops/stage7/model_monitoring_setup.ipynb

```diff
@@ -502,6 +502,86 @@
         "- For the CSV data source, grant both service accounts the \"Storage Object Viewer\" role."
```

Comment:
<p>I think <code>import os</code>is missing here and please remove the output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/902/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>